src/hotspot/share/oops/klass.inline.hpp

```diff
@@ -44,6 +44,10 @@ inline bool Klass::is_non_strong_hidden() const {
 // Iff the class loader (or mirror for non-strong hidden classes) is alive the
 // Klass is considered alive. This is safe to call before the CLD is marked as
 // unloading, and hence during concurrent class unloading.
+// This returns false if the Klass is unloaded, or about to be unloaded because the holder of
+// the CLD is strongly reachable.
```

Comment:
Is that a typo? I would expect being unreachable to lead to unloading?

Reply:
It is a typo thanks.